application/app/src/test/resources/application.yaml

```diff
@@ -44,3 +46,13 @@ cognito:
   userPool.clientId: testUserPoolClientId
   userPool.id: testUserPoolId
   userPool.clientSecret: testUserPoolClientSecret
+
+postcode:
+  mapping:
+    url: https://127.0.0.1:18085/api
```

Comment:
Double check that this is the correct url and port. If we are running the tests in their own container will they be able to see the fuzzy search api/postcode api on local host? Or does it need to be the relative docker network ip?

@jope1 Thoughts?